src/target/cortexm.c

```diff
@@ -227,6 +227,27 @@ static void cortexm_mem_write(target_s *target, target_addr64_t dest, const void
 	adiv5_mem_write(cortex_ap(target), dest, src, len);
 }
 
+bool target_is_cortexm(target_s *target)
+{
+	if ((target == NULL) || (target->priv_free != cortex_priv_free)) {
+		return false;
+	}
+	return true;
+}
+
+uint32_t cortexm_demcr_read(target_s *target)
+{
+	cortexm_priv_s *priv = (cortexm_priv_s *)target->priv;
```

Comment:
`priv` can be made a constant pointer as in `cortexm_priv_s *const priv`, and so can `target` in the same way - let's do that for `const`-correctness.

Similar comment applies to the function below, but including `demcr` itself which can be made `const` too.

Reply:
Alright, it's now as consty as it can be.